reactive/docker.py

```diff
@@ -94,6 +94,16 @@ def upgrade():
         'Holding docker packages at current revision.')
 
 
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    status_set('blocked', 'Series upgrade in progress')
```

Comment:
Seems like this status could easily get overwritten by other handlers. I recommend setting it in an atexit handler or refactoring the charm to use layer-status.